src/pages/authentication/authentication.css

```diff
@@ -1,5 +1,5 @@
 
-.main{
+.authenticate-main{
     background-color: var(--bg-modal);
```

Comment:
Try to avoid large values in absolute it will affect in responsiveness instead use flex or grid it